core/server/worker/src/main/java/alluxio/worker/block/DefaultBlockWorker.java

```diff
@@ -638,6 +638,13 @@ public void closeUfsBlock(long sessionId, long blockId)
           // the sessionId.
           LOG.debug("Invalid worker state while committing block.", e);
         }
+      } else {
+        // When getTempBlockMeta() return null, such as a block readType NO_CACHE writeType THROUGH.
```

Comment:
@xichen01 Would you please show the cause why counter was not be decreased in the commitblock()?